Zhongli.Bot/Modules/HelpModule.cs

```diff
@@ -26,12 +28,18 @@ public async Task HelpAsync()
                 .Select(d => d.Name)
                 .OrderBy(d => d);
 
+            var configuration = new ConfigurationBuilder()
+                .AddUserSecrets<ZhongliConfig>()
+                .Build();
+
+            var prefix = configuration.GetSection(nameof(ZhongliConfig.Debug))[nameof(BotConfig.Prefix)];
```

Comment:
Should check if the configuration is actually debug like the others

Reply:
Yep, totally missed this. Resolved in my latest commit. 

![image](https://user-images.githubusercontent.com/966624/125173696-5f3a1f80-e175-11eb-8e78-fc3ffcf656f9.png)